packages/data-point/lib/reducer-types/reducer-entity-instance/index.js

```diff
@@ -0,0 +1,11 @@
+/* eslint global-require: 0 */
```

Comment:
is this line necessary? I feel like I've removed these lines without getting any linting complaints